src/main/java/shapes/Rectangle.java

```diff
@@ -40,12 +45,24 @@ public void setLength(double length) {
     }
 
     //perimeter of rectangle
+
+    @Override
     double getPerimeter() {
         return 2 * (getLength() + getWidth());
     }
 
+    @Override
+    boolean isInside(double x, double y) {
+        if (Math.abs(x)<length && Math.abs(y)<width) {
```

Comment:
This could be simplified:
The result of `if` statement is already what you're looking for, so no need to do `if .. else`.
It can be just `return Math.abs(x) < length && Math.abs(y) < width;`